src/managers/config.py

```diff
@@ -678,6 +679,10 @@ def server_properties(self) -> list[str]:
             if self.state.kraft_mode == False:  # noqa: E712
                 properties += self.zookeeper_tls_properties
 
+        if self.state.runs_balancer or self.state.peer_cluster_orchestrator_relation:
```

Comment:
**todo**: instead of peer_cluster_orchestrator, check that balancer is in peer_cluster roles

Reply:
Fixed in 0b20a815e1fba778258e93c41b3be681347f01e6